engine/language-server/src/main/resources/application.conf

```diff
@@ -8,6 +8,7 @@ logging-service.logger {
   akka.event = error
   akka.io = error
   akka.stream = error
-  slick = error
+  slick.jdbc.JdbcBackend.statement = error # log SQL queries on debug level
```

Comment:
I decided not to enable the debug logs for `slick.jdbc.JdbcBackend.statement` by default because it produces a lot of output. I think in general, we're not interested in SQL queries unless we're debugging some specific issue.

Reply:
Makes sense now. Just from the comment I assumed this should have been `= debug`.